lib/cisco_node_utils/bgp_af.rb

```diff
@@ -853,59 +588,16 @@ def method_missing(*args)
       end
     end
 
-    ##########################################
-    # All of this can be replaced by the method_missing metafunction
-    #
-    #     def default_information_originate=(state)
-    #     def default_default_information_originate
-    #     def next_hop_route_map
-    #     def default_next_hop_route_map
-    #     def additional_paths_send=(state)
-    #     def additional_paths_send
-    #     def default_additional_paths_send
-    #     def additional_paths_receive
-    #     def additional_paths_receive=(state)
-    #     def default_additional_paths_receive
-    #     def additional_paths_install
-    #     def additional_paths_install=(state)
-    #     def default_additional_paths_install
-    #     def additional_paths_selection
-    #     def default_additional_paths_selection
-    #     def default_advertise_l2vpn_evpn
-    #     def dampen_igp_metric
-    #     def default_dampen_igp_metric
-    #     def default_dampening_state
-    #     def default_dampening_max_suppress_time
-    #     def default_dampening_half_time
-    #     def default_dampening_reuse_time
-    #     def default_dampening_routemap
-    #     def default_dampening_suppress_time
-    #     def default_distance_ebgp
-    #     def default_distance_ibgp
-    #     def default_distance_local
-    #     def default_default_metric
-    #     def default_inject_map
-    #     def maximum_paths
-    #     def default_maximum_paths
-    #     def maximum_paths_ibgp
-    #     def default_maximum_paths_ibgp
-    #     def default_networks
-    #     def default_redistribute
-    #     def route_target_both_auto
-    #     def default_route_target_both_auto
-    #     def route_target_both_auto_evpn
-    #     def default_route_target_both_auto_evpn
-    #     def default_route_target_export
-    #     def default_route_target_export_evpn
-    #     def default_route_target_import
-    #     def default_route_target_import_evpn
-    #     def suppress_inactive
-    #     def suppress_inactive=(state)
-    #     def default_suppress_inactive
-    #     def table_map
-    #     def table_map_filter
-    #     def default_table_map
-    #     def default_table_map_filter
-    ##########################################
+    # Is the given name available in the YAML?
+    def responds_to?(name)
```

Comment:
The relevant Ruby API is `respond_to?` not `responds_to?`. See http://technicalpickles.com/posts/using-method_missing-and-respond_to-to-create-dynamic-methods/ for a pretty good walkthrough.